scripts/Palindrome.py

```diff
@@ -6,16 +6,14 @@ def is_palindrome(text):
     left_side_idx = 0
     right_side_idx = len(text) - 1
 
-    is_same_letter = True
-    while left_side_idx != right_side_idx and is_same_letter:
+    while left_side_idx != right_side_idx and result:
```

Comment:
and result - sounds not good.
Here is it better to use some meaningful var name (like you deleted before)